lib/json/static.nit

```diff
@@ -31,6 +31,9 @@ private import json_lexer
 interface Jsonable
 	# Encode `self` in JSON.
 	#
+	# This is a recursive method which can be refined by any subclasses.
```

Comment:
Right, that is (almost) the meaning of the `abstract` keyword, but it's not the goal of this doc to explain the `abstract` keyword. I expect reader to have basic programming knowledge.

It says that **any** subclass can refine the behavior, it is in opposition with methods in which we discourage subclasses to refine the behavior.